src/components/VDataIterator/VDataIterator.js

```diff
@@ -0,0 +1,119 @@
+require('../../stylus/components/_data-iterator.styl')
+
+import DataIterable from '../../mixins/data-iterable'
+
+export default {
+  name: 'v-data-iterator',
+
+  mixins: [DataIterable],
+
+  inheritAttrs: false,
+
+  props: {
+    tag: {
```

Comment:
Perhaps change this to `content-tag`, and add explicit `content-props` prop (Object) to be used with `v-bind=` on content element?